hicommonwealth/commonwealth
# #6512 - Update Editor Component UI - Toolbar Lists

Allows user to create a list without selecting text first. 

## Link to Issue
Closes: #6512

## Description of Changes
- allows users to add lists without selecting text

## Test Plan
- visit create thread page / create comment
- Highlight text, make sure all list types work
- Make sure toggling lists work
- Make sure switching lists work as intended 
- Make sure numbered lists are in order 

## Other considerations

Keep in mind there is a seperate ticket for continuing a list via the enter button: https://github.com/hicommonwealth/commonwealth/issues/6683
